### PR TITLE
[NEW] Block enable option for apps limit

### PR DESCRIPTION
**Pull request by tiagoevanp**
## Proposed changes (including videos or screenshots)

image

## Further comments

MKP-215

### CODE REVIEW

**Comment by tiagoevanp** [comment]
https://github.com/RocketChat/fuselage/pull/981 : Need this as a dependency!

File: apps/meteor/client/views/marketplace/AppMenu.js [hunk 1]
```diff
@@ -335,10 +335,10 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 				isAdminUser && {
 					subscribe: {
 						label: (
-							<Option>
+							<Box>
```
Comment (ggazzo) [inline apps/meteor/client/views/marketplace/AppMenu.js:338]: please check how it goes with <Fragment/> instead of <Box/>

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/marketplace/AppMenu.js (hunk 1 above)
Comment (rique223) [reply apps/meteor/client/views/marketplace/AppMenu.js:338]: With what?

Comment (d-gubert) [reply apps/meteor/client/views/marketplace/AppMenu.js:338]: I've updated his comment, it was missing the backticks on the tags and GitHub was trying to render it I guess :eyes:

Comment (rique223) [reply apps/meteor/client/views/marketplace/AppMenu.js]: Solved

**Review by rique223 — APPROVED** [review]
Front end wise it looks good.

**Pull request merged**